src/plugins/discover/public/application/components/chart/chart.tsx

```diff
@@ -95,15 +95,15 @@ export const DiscoverChart = ({
   );
 
   const toggleLabel = i18n.translate('queryEditor.collapse', {
-    defaultMessage: 'Toggle query editor',
+    defaultMessage: 'Toggle discover histogram',
```

Comment:
I think you can just say `Toggle histogram` already in discover page would be redudant